README.md

```diff
@@ -1,2 +1,69 @@
+- requires multiple parties/working groups/owners participating to complete (ex. auto mTLS)
+- will be graduating from one stage to another (ex. alpha to beta, beta to GA)
+- needs significant effort or changes Istio in a significant way (ex. something that would take 10 person-weeks to implement, introduce or redesign a system component, or introduces API changes)
+- impacts the UX or operation of Istio substantially such that engineers using Istio will need retraining
+- users will notice and come to rely on
+
+It is unlikely an enhancement if it is:
+- fixing a flaky test
+- refactoring code
+- performance improvements, which are only visible to users as faster API operations, or faster control loops
+- adding error messages or events
+
+If you are not sure, ask someone in the working group where you initially circulated the idea.
+
+## When to Create a New Enhancement Issue
+
+Create an issue here once you:
+- have circulated your idea to see if there is interest
+   - through Community Meetings, working group meetings, discuss.istio.io lists, or an issue in github.com/istio/istio
+- (optionally) have done a prototype in your own fork
+- (optionally) have identified people who agree to work on the enhancement
+  - many enhancements will take several releases to progress through Alpha, Beta, and Stable stages
+  - you and your team should be prepared to work on the approx. 9mo - 1 year that it takes to progress to Stable status
+- (optionally) are ready to be the project-manager for the enhancement
+
+## Why are Enhancements Tracked
+
+Once users adopt an enhancement, they expect to use it for an extended period of time. Therefore, we hold new enhancements to a
+high standard of conceptual integrity and require consistency with other parts of the system, thorough testing, and complete
+documentation. As the project grows no single person can track whether all those requirements are met. The development
+of an enhancement often spans three stages: Alpha, Beta, and Stable; Enhancement Tracking Issues provide a
+checklist that allows for different approvers for different aspects, and ensures that nothing is forgotten across the
+development lifetime of an enhancement.
+
+## When to Comment on an Enhancement Issue
+
+Please comment on the enhancement issue to:
+- request a review or clarification on the process
+- update status of the enhancement effort
+- link to relevant issues in other repos
+
+Please do not comment on the enhancement issue to:
+- discuss a detail of the design, code or docs. Use a linked-to-issue or PR for that
+
+## Lifecycle of an Ehancement Issue
+
+- An enhancement issue must be presented to corresponding working group(s) meeting first, with a link to the RFC document.  If the enhancement spans more than one working groups, it must be presented to each of the working group to secure approvals.
```

Comment:
Do we have a process for ensuring that all impacted WGs are looped in? Often, impact is not well-understood. Networking changes can have dramatic impact on telemetry without networking teams realizing, etc.

With the fear of creating more process, I might suggest that each IEP *must* contain a direct assessment of impact across all WG-level concerns, and that those sections be reviewed by each impacted WG.

Reply:
It seems reasonable, although (as you said) often the impact is not well understood until the implementation is nearing completion. I think we need to trust our working groups to do the right thing, and when they create a crater, to reach out to the impacted working group and determine a solution that is reasonable for both parties.

Reply:
My concern, based on experience, is that waiting until near completion to check on impact puts a weird onus on the impacted party to (scramble to) find a way to make something work (as considerable effort has already gone into something and plans/promises have already been made).

If we are going to formalize the planning and roadmap process, I would like us to at least attempt to address the need for fully-articulating impact. We won't always get it right. But, my idealistic hope is that if we, say, have required sections like "Telemetry Collection Impact" and "Environmental Support" in whatever form we adopt, we will catch at least more potential issues than we miss.

Maybe that fits better in the Design Docs ? The relationship between all three is fuzzy to me at the moment.